apps/frontend/src/views/Results.vue

```diff
@@ -147,7 +147,68 @@
         <v-row>
           <v-col xs-12>
             <v-card elevation="2">
-              <v-card-title>Tree Map</v-card-title>
+              <v-card-title class="d-flex align-center">
+                <span>Tree Map</span>
+                <v-spacer />
+                <v-btn class="mr-2" icon small @click="isActiveDialog = true">
+                  <v-icon
+                    b-tooltip.hover
+                    title="Tree Map Legend"
+                    color="primary"
+                  >
+                    mdi-information-outline
+                  </v-icon>
+                </v-btn>
+                <v-dialog v-model="isActiveDialog" width="500">
+                  <v-card>
+                    <v-card-title>Tree Map Legend</v-card-title>
+                    <v-card-text class="text-h7">
```

Comment:
If the css mechanism works, then I also want us to put the text on the js side that is then templated in as opposed to hard coding into the html.